src/include/hwinfo/system_settings_ui.rb

```diff
@@ -58,7 +58,9 @@ def ReadSystemSettingsDialog
       # to avoid of the very long starting time of the yast module
       # because the Storage module (which is imported by the Bootloader (imported by the SystemSettings module))
       # has a Read() function call in its constructor.
-      SystemSettings.Read
+      system_settings = SystemSettings.Read
+      return :abort unless system_settings
```

Comment:
I would comment here that we simply return `:abort` without any message because `SystemSettings.Read` already reports the problem to the user. It's is obvious in the context of the bug/Trello card, but it may not be that obvious after some time.

Reply:
You're completely right @ancorgs

Thanks so much for _thinking aloud_ :)